### ISSUE TITLE
p1_HierarchicalAttention_model.py中关于维度问题

### ISSUE THREAD

**Issue by searchlink** [brightmart/text_classification#90]
inference()中，
对于input_x = tf.split(self.input_x, self.num_sentences,axis=1),
其shape为：num_sentences个[None,self.sequence_length/num_sentences]
input_x = tf.stack(input_x, axis=1)
其shape为：[None, num_sentences，self.sequence_length/num_sentences]
self.embedded_words = tf.nn.embedding_lookup(self.Embedding,input_x)
其shape不应该是[None,self.num_sentences,self.sequence_length/num_sentences, self.embed_size]吗？
为什么我看注释是[None,num_sentences,sentence_length,embed_size]

**Comment by brightmart** [comment]
you are right. comment may not correct.

**Comment by searchlink** [comment]
多谢~
